nginx/sites-enabled/hashbang-http

```diff
@@ -0,0 +1,13 @@
+# Hashbang server configuration
+# ref: https://gist.github.com/RyanSquared/e59c7e274abef06e63b0e47e46997b00
+#
+server {
+	listen 80;
+	listen [::]:80;
+
+	server_name "~^(?<user>[a-zA-Z0-9]+)\.(?<server>\w{2}\d{1})\.hashbang\.sh$";
+
+	location / {
+		proxy_pass http://unix:/var/run/http/$user-http.sock;
```

Comment:
How did we come to pick this directory?
I thought we might use `/run/user/$UID/hashbang-http/user.sock` or similar. but that would require mapping from user to uid...